package.json

```
@@ -35,8 +35,7 @@
     "eslint": "eslint ./",
     "pack": "npm install && npm run build && npm run clean-dist && npm run dist",
     "pack:win": "cmd /c npm install && npm run install-electron-dependencies && npm run clean-build && npm run copy-files && npm run build-prod && npm run build-electron && npm run clean-dist && npm run dist:win",
-    "storybook": "start-storybook -p 6006 -s ./src/",
-    "build-storybook": "build-storybook",
+    "storybook": "cd ./storybook/; npm run storybook",
```

It would be nice to continue in the same folder after stopping storybook
```suggestion
    "storybook": "cd ./storybook/; npm run storybook; cd ..",
```